cad/LibreCAD/Portfile

```diff
@@ -2,10 +2,44 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
-PortGroup           qmake 1.0
+PortGroup           qmake5 1.0
+PortGroup           cxx11 1.1
+
+name                LibreCAD
+subport             LibreCAD-devel {}
+
+qt5.depends_component qtsvg qttranslations qttools
+
+if {${subport} eq "LibreCAD"} {
```

Comment:
Is there a reason to use `if {${subport} eq "LibreCAD"}` instead just `subport ${name}` which is easier to parse?